l10n-screenshots.sh

```diff
@@ -13,8 +13,8 @@ PROJECT_DIR="$(get_abs_path $CURRENT_DIR/../../../..)"
 
 
 if [ -d l10n-screenshots ]; then
-  echo "The l10n-screenshots directory already exists. You decide."
-  exit 1
+    echo "The l10n-screenshots directory already exists. You decide."
+    exit 1
```

Comment:
nit: extra space? also above and some lines below

Reply:
yeah, I aligned every if statement to use 4-space-indents. The rest of the file seems to use this convention. I'm fine having everything aligned at 2 spaces too 🙂